packages/api/src/database/migrations-manager.ts

```diff
@@ -22,10 +22,25 @@ export class MigrationsManager {
     try {
       logger.info(`🆙 Starting migrations for: ${community.hostname}`);
 
-      // Connect to the community db
-      const mongooseConn = await mongoose.connect(dbUrlCommunity(community), {
-        useNewUrlParser: true,
-      } as ConnectOptions);
+      let mongooseConn: Connection;
+      try {
+        // Connect to the community db https://stackoverflow.com/a/65205700/4650625
+        const mongooseInstance = await mongoose.connect(
+          dbUrlCommunity(community),
+          {
+            useNewUrlParser: true,
+          } as ConnectOptions,
+        );
+        mongooseConn = mongooseInstance.connection;
```

Comment:
@kristoferlund 
I was passing mongoose instance instead of connection in this part of code


I fixed it and if you run the API you would see all migrations executed successfully